src/dataset-builder/TestConstants.ts

```diff
@@ -175,3 +175,128 @@ const dummyConcepts = [
 export const dummyGetConceptForId = (id: number): Concept => {
   return _.find({ id }, dummyConcepts)!;
 };
+
+interface ChartLabel {
+  short: string;
+  long?: string;
+}
+interface ChartSeries {
+  name?: string;
+  data: number[];
+}
+
+export interface CohortDemographics {
+  categories: ChartLabel[];
+  series: ChartSeries[];
+  title: string;
+  yTitle: string;
+  height: string;
+  legendEnabled: boolean;
+  showSeriesName: boolean;
+}
+
+export function generateCohortAgeData(series) {
```

Comment:
This function doesn't seem like test data to me, since its the chart definition. I'd move this into the component.